R/dash.R

```diff
@@ -1961,24 +1965,6 @@ Dash <- R6::R6Class(
         private$.index <- private$template_index
       }
 
-      # define the react-entry-point
-      app_entry <- "<div id='react-entry-point'><div class='_dash-loading'>Loading...</div></div>"
-      # define the dash default config key
-      config <- sprintf("<script id='_dash-config' type='application/json'> %s </script>", to_JSON(self$config))
-
-      if (is.null(private$name))
-        private$name <- 'Dash'
-
-      if (!is.null(private$custom_index)) {
-        string_index <- glue::glue(private$custom_index, .open = "{%", .close = "%}")
-
-        private$.index <- string_index
-      }
-
-      else if (length(private$template_index) == 1) {
-        private$.index <- private$template_index
-      }
-
```

Comment:
Nice catch -- this is all dead code from before the index customization PR was merged, eh?

Reply:
I think it was a copy/paste error during the index customization PR, it's just the same codeblock repeated as the one a few lines above it. Safe to cut :scissors: